AsciiDoc RN: various changes

KucharczykL opened the pull request:
Inspired by https://bugzilla.suse.com/show_bug.cgi?id=1188302.

sknorr commented on asciidoc-release-notes/adoc/about-documentation.adoc:
```diff
@@ -31,4 +36,4 @@ endif::[]
 endif::[]
-* Find a collection of White Papers in the {product} Resource Library at https://www.suse.com/products/server#resources.
\ No newline at end of file
+* Find a collection of White Papers in the {product} Resource Library at https://www.suse.com/products/server#resources.
```
Are you sure you want this link in literal form? I would suggest adding an AsciiDoc attribute like `{reslib-url}` to product-generic.adoc for this.

KucharczykL replied:
That is not a new line in this commit. It's just that the .editorconfig configuration adds a newline at the end of files and there wasn't one before. If you disable whitespace differences it's not highlighted anymore.

KucharczykL replied:
It's a good point, though. If you don't mind I will change it in this otherwise unrelated PR.

sknorr replied:
Thanks for fixing!

>If you disable whitespace differences it's not highlighted anymore.

I didn't realize that's an option in GH. (I found it, behind the cogwheel.)

sknorr reviewed (commented):
Thanks - looks good except ...

KucharczykL commented:
I have replaced the resource library URL with attribute and added another release note change that should be propagated (see https://bugzilla.suse.com/show_bug.cgi?id=1189989).

sknorr commented:
Thanks! Merged.
